docs/source/developers/index.md

```diff
@@ -392,6 +392,42 @@ custom = "custom_package:CustomChatHandler"
 Then, install your package so that Jupyter AI adds custom chat handlers
 to the existing chat handlers.
 
+## Streaming output
```

Comment:
This section should focus on "Streaming output from slash commands" for developers. This section should basically instruct developers streaming from custom slash commands to use the new `stream_reply()` method available in the base class, as of v2.26.0. 

- There doesn't need to be any mention of `@file`, since right now, `@file` does not work when embedded in a slash command. Only `DefaultChatHandler` calls the context providers when `@file` is passed.

- There should be an example on how to construct a runnable using LCEL in `self.create_llm_chain()` method.

- There should also be an example on how to use call `self.stream_reply()` within `self.process_message()` to stream a reply from a runnable bound to `self.llm_chain`, set by `self.create_llm_chain()`.

For the examples, it would probably be best to borrow the implementation in `/fix` here: https://github.com/jupyterlab/jupyter-ai/pull/1048

I'll review that now so it can be merged.